content/sensu-go/6.1/operations/control-access/rbac.md

```diff
@@ -305,6 +305,9 @@ Passwords must have at least eight characters.
 sensuctl user create alice --password='password' --groups=ops,dev
 {{< /code >}}
 
+You can create any number of users, each with their own passwords.
+Any agent user must belong to the `system:agent` group.
```

Comment:
Actually it's most important that the agent user be granted the `system:agent` role. 

By default the `agent` user belongs to the `system:agent` group and the `system:agent` cluster role binding grants the `system:agent` cluster role to the members of the `system:agent` group.

Starting from a default RBAC configuration, adding a user to the `system:agent` group is the recommended way to grant the permissions an agent needs to report events into any namespace.

Reply:
We have a section here on default roles but seem to be lacking one for default role bindings and default cluster role bindings, as well as the default cluster roles themselves. Documenting these will help to complete the picture with regard to how agent authorization works.

Reply:
The "default roles" list only listed cluster roles, so I added the one default role. I also added a separate table for the default role bindings and cluster role bindings in that section below.